packages/wallet/src/services/TimeSettingsManualTracker.ts

```diff
@@ -0,0 +1,45 @@
+import { BehaviorSubject } from 'rxjs';
```

Comment:
I omitted tests for this file intentionally because:
- It is only a wrapper over coldObservable.
- Testing it would mean doing similar mocks as in the coldObservable tests, which would dup the work there.
- An alternative would be to mock the coldObservable itself, leaving nothing to test but a BehaviourSubject.